packages/vite/src/template-tag.ts

```diff
@@ -19,26 +19,32 @@ export function templateTag({ inline_source_map } = { inline_source_map: false }
       if (options.custom?.embroider?.isExtensionSearch) {
         return null;
       }
-      let resolution = await this.resolve(id, importer, {
-        skipSelf: true,
-      });
-      if (resolution) {
-        return resolution;
-      }
-      for (let candidate of candidates(id)) {
-        resolution = await this.resolve(candidate, importer, {
+      if (id.endsWith('.gjs') || id.endsWith('.gts')) {
```

Comment:
This condition doesn't make sense. If the request already has an explicit `.gjs` extension, the default resolving will handle it fine and this plugin will need no custom behavior in `resolveId`.